builds/android/res/values-de/strings.xml

```diff
@@ -31,7 +32,49 @@
 	<string name="not_valid_game">$PATH ist kein gültiges Spiel</string>
 	<string name="accessing_configuration_failed">Konfiguration von $PATH konnte nicht gelesen werden.</string>
 	<string name="select_game_region">Spielregion auswählen</string>
+	<string name="change_the_layout">Layout ändern</string>
+	<string name="choose_layout">Layout auswählen</string>
 	<string name="unknown_region">Unbekannte Region</string>
-	<string name="region_modification_success">Region in $NAME geändert</string>
 	<string name="region_modification_failed">Ändern der Region fehlgeschlagen</string>
+	<string name="refresh">Aktualisieren</string>
+	<string name="change_default_mapping">Standard-Tastenlayout ändern</string>
+	<string name="how_to_use_easy_rpg">Wie verwende ich EasyRPG</string>
+	<string name="how_to_use_easy_rpg_explanation">
+Installation:\n
+\t1. Platziere die Spiele in /easyrpg/games im Hauptverzeichnis deines Geräts.\n
+\t2. Platziere das RTP in /easyrpg/rtp/2000 für das RPG Maker 2000 RTP bzw. in /easyrpg/rtp/2003 für das RPG Maker 2003 RTP.\n
+\n
+Wenn Spiele nicht erkannt werden und dein Gerät hat mehr als einen Speicher (z.B. MicroSD) zur Verfügung, versuche den easyrpg-Ordner auf den anderen Speicher zu kopieren und drücke "Aktualisieren".\n
+\n
+Tipp: Damit Spiele korrekt erkannt werden, müssen sie entpackt sein und dürfen NICHT in Unterordnern liegen, z.B..:\n
+\t/easyrpg/games/MeinSpiel/Data/RPG_RT.ldb ist FALSCH.\n
+\t/easyrpg/games/MeinSpiel/Data/GameOver/GameOver.png ist FALSCH.\n
+\t/easyrpg/games/MeinSpiel/RPG_RT.ldb ist RICHTIG.\n
+\t/easyrpg/games/MeinSpiel/GameOver/GameOver.png ist RICHTIG.\n
+\n
+Spezielle Funktionen:\n
+\tIm Einstellungsmenü kannst du Tasten verschieben und neue hinzufügen.\n
+	</string>
+	
+	<!-- Settings -->
+	<string name="settings">Einstellungen</string>
+	<string name="enable_vibration">Vibrationen aktivieren</string>
+	<string name="vibrate_when_sliding_direction">Vibrieren, wenn in eine andere Bewegungsrichtung gewischt wird</string>
+	<string name="input_layout_transparency">Layout-Transparenz:</string>
+	<string name="ignore_size_settings">Verwende folgende Tastengröße anstatt der Standardgröße:</string>
+	
+	<!-- InputLayout Managing activity -->
+	<string name="manage_input_layouts">Eingabe-Layouts verwalten:</string>
+	<string name="add_an_input_layout">Eingabe-Layout hinzufügen</string>
+	<string name="set_as_default">Als Standard festlegen</string>
+	<string name="edit_name">Umbenennen</string>
+	<string name="edit_layout">Layout bearbeiten</string>
+	<string name="delete">Löschen</string>
+	<string name="default_layout">Standard</string>
+	
+	<!-- ButtonMapping' activity-->
+	<string name="add_a_button">Taste hinzufügen</string>
+	<string name="reset_button_mapping">Tastenbelegung zurücksetzen</string>
+	<string name="exit_without_saving">Ohne speichern beenden</string>
+	<string name="save_and_quit">Speichern und Beenden</string>
```

Comment:
I wonder if the different casing of `speichern` and `beenden` is okay. Maybe should be consistent.